tests/e2e/bcd_consumer_integration_test.go

```diff
@@ -1170,6 +1200,7 @@ func (s *BCDConsumerIntegrationTestSuite) createVerifyConsumerFP() (*bstypes.Fin
 // helper function: initBabylonController initializes the Babylon controller with the default configuration.
 func (s *BCDConsumerIntegrationTestSuite) initBabylonController() error {
 	cfg := config.DefaultBabylonConfig()
+	cfg.BlockTimeout = 10 * time.Minute
```

Comment:
Let's move this (or a positive value) to the default config.

I have a couple questions:

1. What's the difference between `Timeout` and `BlockTimeout`? Is the first a connection timeout or so?
2. Why a zero value of block timeout didn't caused issues before?

Reply:
1. This needs to be fixed from Babylon side, i.e., by https://github.com/babylonlabs-io/babylon/pull/731
2. This was introduced by a recent PR in Babylon AFAIK